### PR TITLE
Local Test Fixes

### PR DESCRIPTION
**Pull request by benedikt-brandtner-bikemap**
This PR fixes local tests affected by the changes introduced in #1440 #1444 #1361 #1501

### CODE REVIEW

File: layers/transportation/update_transportation_merge.sql [hunk 1]
```diff
@@ -483,7 +483,7 @@ BEGIN
         WHERE
             (highway = 'motorway'
             OR construction = 'motorway'
-            -- Allow trunk roads that are part of a nation's most important route network to show at z4
+            -- Allow trunk roads that are part of a nation's most important route network to show at z5
             OR (highway = 'trunk' AND osm_national_network(network))
```
Comment (ZeLonewolf) [inline layers/transportation/update_transportation_merge.sql:487]: Is this `highway = 'trunk'` necessary given the change at line 525?

**Comment by ZeLonewolf** [comment]
This PR reverses #1444 which corrected a _bug_ introduced in #1440. It was never anyone's intention to show all motorways at z4, only the most important ones. So far "most important" has only been defined for the US and Canada, but the intent is that similar top-level networks in other countries would be similarly defined for visibility at z4. Unless there is a specific client demand for z4 motorways in general, I don't think it makes sense to render them to the tiles as a universal rule. Of course the client can always filter them out at style time.

**Comment by benedikt-brandtner-bikemap** [comment]
Hey, oh sorry so the intention of #1444 was to remove all motorways from `osm_transportation_merge_linestring_gen_z4`?

Because in the currently released version 3.14 this table contains all motorways independent of their network tags.
Then #1440 added trunks with the mentioned network tags and #1444 removed those trunks again but also removed all motorways and i thought that was not intentional.
But if it was then of course i'll revert https://github.com/openmaptiles/openmaptiles/pull/1505/commits/83589c25f587143e98e69fa91f86646a667eea6e

File: layers/transportation/update_transportation_merge.sql (hunk 1 above)
Comment (benedikt-brandtner-bikemap) [reply layers/transportation/update_transportation_merge.sql:487]: That is correct if we add all important networks independent of their highway/construction tags to z4 anyways we should do it here as well.

**Comment by ZeLonewolf** [comment]
The intent of the transportation layer at zoom 4 is to render an unlabeled transportation network that shows the overall shape of the road network without excessive detail. Including "any motorway" at that zoom would include many motorway spurs and islands that aren't desirable to see at that zoom.

Some of the discussion in ZeLonewolf/openstreetmap-americana#565 might be helpful to understand what this lowest-zoom rendering is trying to achieve.  Also #1444 has some of the metrics, which include a 15% tile size difference at z5 (!!).

Adding test cases of course is a good idea (and would have prevented me from making the mistake in the first class!)

**Comment by benedikt-brandtner-bikemap** [comment]
Alright, have removed the commit and ammended the tests

**Comment by benedikt-brandtner-bikemap** [comment]
Have also removed this branch as a base for all the other PRs and rebased them ontop of master.

**Comment by ZeLonewolf** [comment]
Good work.  I dropped a note in Slack asking for a maintainer to run workflows.

**Comment by benedikt-brandtner-bikemap** [comment]
thanks a lot :)

**Comment by TomPohys** [comment]
Thanks a lot, @benedikt-brandtner-bikemap! Could you please also edit the "explanation text" for this PR? Thanks!

**Comment by benedikt-brandtner-bikemap** [comment]
Hey, have updated the title and description

**Comment by TomPohys** [comment]
Thanks a lot!

**Pull request merged**